pkg/database/sqlite/config.go

```diff
@@ -1,17 +1,44 @@
 package sqlite
 
+import (
+	"os"
+	"path/filepath"
+)
+
 // Config implements database.Configurator interface and
 // handles the default configuration parameters of the sqlite database.
 type Config struct {
 	Path string
+	Home string
 }
 
 func (c *Config) SetDefaults() {
-	if c.Path == "" {
-		c.Path = "storage.db"
+	if c.Path != "" {
+		return
 	}
+
+	c.Path = DefaultPath(c.Home)
 }
 
 func (c *Config) Validate() error {
 	return nil
 }
+
+// DefaultPath returns the default sqlite path for the given home directory.
+func DefaultPath(home string) string {
+	if home == "" {
+		home = os.Getenv("TERRALIST_HOME")
+	}
+
+	if home == "" {
+		if userHome, err := os.UserHomeDir(); err == nil {
+			home = userHome
+		}
+	}
+
+	if home == "" {
+		home = "."
+	}
```

Comment:
This fallback might have sense in the context of the user manually specifying `home: ""`, but I would consider forcefully falling back to the flag default in this scenario, as I believe it currently permits this behavior, instead of treating it on a case-by-case basis.

See https://github.com/terralist/terralist/blob/master/pkg/cli/path_flag.go#L51-L53.